.gitattributes

```diff
@@ -0,0 +1,9 @@
+# Set the default behavior, in case people don't have core.autocrlf set.
+* text=auto
+
+# Set our files to use lf, so our prettify will always be happy
```

Comment:
Fixes the files switching line ending on windows after checkout and therefore showing up as changed after running prettify.